src/test/java/api/requests/HoldShelfClearanceReportTests.java

```diff
@@ -590,7 +590,37 @@ private void verifyResponse(InventoryItemResource item,
 
     JsonObject itemJson = requestJson.getJsonObject(ITEM_KEY);
     assertThat(itemJson.getString(BARCODE_KEY), is(item.getBarcode()));
-    String callNumber = item.getHoldingsRecord().getJson().getString(CALL_NUMBER_KEY);
-    assertThat(itemJson.getString(CALL_NUMBER_KEY), is(callNumber));
+
+    JsonObject expectedCallNumberComponents = item.getJson()
+      .getJsonObject("effectiveCallNumberComponents");
+    assertThat(itemJson.getString(CALL_NUMBER_KEY),
+      is(expectedCallNumberComponents.getString("callNumber")));
+
+    JsonObject actualCallNumberComponents = itemJson.getJsonObject("callNumberComponents");
```

Comment:
I personally prefer that tests for APIs that respond with more than one record, check that the values for different records are different.

This might be too big a consideration, given how these tests are currently written.

Reply:
The purpose of the test mainly to verify that the properties are got populated in response. I assume that derivation logic of the properties have already been covered separately.

Reply:
We've had bugs in the past where two records in the same response have had incorrect properties because records have been linked up incorrectly. 

Which led me to want tests with at least two records that have different state. I imagine these tests only respond with single items / requests for response?

Reply:
Ok, I've adjusted `multipleClosedPickupExpiredRequest` test, now it asserts returned requests.